[2023-07-22T05:54:10.910+0000] {taskinstance.py:1165} INFO - Dependencies all met for <TaskInstance: user_processing.create_table scheduled__2023-07-12T00:00:00+00:00 [queued]>
[2023-07-22T05:54:10.954+0000] {taskinstance.py:1165} INFO - Dependencies all met for <TaskInstance: user_processing.create_table scheduled__2023-07-12T00:00:00+00:00 [queued]>
[2023-07-22T05:54:10.969+0000] {taskinstance.py:1362} INFO - 
--------------------------------------------------------------------------------
[2023-07-22T05:54:10.973+0000] {taskinstance.py:1363} INFO - Starting attempt 1 of 1
[2023-07-22T05:54:10.985+0000] {taskinstance.py:1364} INFO - 
--------------------------------------------------------------------------------
[2023-07-22T05:54:11.176+0000] {taskinstance.py:1383} INFO - Executing <Task(PostgresOperator): create_table> on 2023-07-12 00:00:00+00:00
[2023-07-22T05:54:11.246+0000] {standard_task_runner.py:55} INFO - Started process 5149 to run task
[2023-07-22T05:54:11.356+0000] {standard_task_runner.py:82} INFO - Running: ['***', 'tasks', 'run', 'user_processing', 'create_table', 'scheduled__2023-07-12T00:00:00+00:00', '--job-id', '37', '--raw', '--subdir', 'DAGS_FOLDER/user_processing.py', '--cfg-path', '/tmp/tmpsxbjx1d9']
[2023-07-22T05:54:11.388+0000] {standard_task_runner.py:83} INFO - Job 37: Subtask create_table
[2023-07-22T05:54:11.840+0000] {task_command.py:376} INFO - Running <TaskInstance: user_processing.create_table scheduled__2023-07-12T00:00:00+00:00 [running]> on host ef71d3f52879
[2023-07-22T05:54:12.138+0000] {taskinstance.py:1592} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=user_processing
AIRFLOW_CTX_TASK_ID=create_table
AIRFLOW_CTX_EXECUTION_DATE=2023-07-12T00:00:00+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2023-07-12T00:00:00+00:00
[2023-07-22T05:54:12.192+0000] {base.py:71} INFO - Using connection ID 'postgres' for task execution.
[2023-07-22T05:54:12.214+0000] {sql.py:315} INFO - Running statement: 
            CREATE TABLE IF NOT EXISTS users (
                firstname TEXT NOT NULL,
                lastname TEXT NOT NULL,
                country TEXT NOT NULL,
                username TEXT NOT NULL,
                password TEXT NOT NULL,
                email TEXT NOT NULL
            );
        , parameters: None
[2023-07-22T05:54:12.226+0000] {postgres.py:96} INFO - NOTICE:  relation "users" already exists, skipping

[2023-07-22T05:54:12.245+0000] {taskinstance.py:1406} INFO - Marking task as SUCCESS. dag_id=user_processing, task_id=create_table, execution_date=20230712T000000, start_date=20230722T055410, end_date=20230722T055412
[2023-07-22T05:54:12.367+0000] {local_task_job.py:164} INFO - Task exited with return code 0
[2023-07-22T05:54:12.490+0000] {local_task_job.py:273} INFO - 1 downstream tasks scheduled from follow-on schedule check
[2023-07-22T05:56:53.184+0000] {taskinstance.py:1165} INFO - Dependencies all met for <TaskInstance: user_processing.create_table scheduled__2023-07-12T00:00:00+00:00 [queued]>
[2023-07-22T05:56:53.226+0000] {taskinstance.py:1165} INFO - Dependencies all met for <TaskInstance: user_processing.create_table scheduled__2023-07-12T00:00:00+00:00 [queued]>
[2023-07-22T05:56:53.226+0000] {taskinstance.py:1362} INFO - 
--------------------------------------------------------------------------------
[2023-07-22T05:56:53.227+0000] {taskinstance.py:1363} INFO - Starting attempt 1 of 1
[2023-07-22T05:56:53.227+0000] {taskinstance.py:1364} INFO - 
--------------------------------------------------------------------------------
[2023-07-22T05:56:53.267+0000] {taskinstance.py:1383} INFO - Executing <Task(PostgresOperator): create_table> on 2023-07-12 00:00:00+00:00
[2023-07-22T05:56:53.298+0000] {standard_task_runner.py:55} INFO - Started process 5442 to run task
[2023-07-22T05:56:53.366+0000] {standard_task_runner.py:82} INFO - Running: ['***', 'tasks', 'run', 'user_processing', 'create_table', 'scheduled__2023-07-12T00:00:00+00:00', '--job-id', '96', '--raw', '--subdir', 'DAGS_FOLDER/user_processing.py', '--cfg-path', '/tmp/tmpsxh2yi49']
[2023-07-22T05:56:53.395+0000] {standard_task_runner.py:83} INFO - Job 96: Subtask create_table
[2023-07-22T05:56:53.794+0000] {task_command.py:376} INFO - Running <TaskInstance: user_processing.create_table scheduled__2023-07-12T00:00:00+00:00 [running]> on host ef71d3f52879
[2023-07-22T05:56:53.995+0000] {taskinstance.py:1592} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=user_processing
AIRFLOW_CTX_TASK_ID=create_table
AIRFLOW_CTX_EXECUTION_DATE=2023-07-12T00:00:00+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2023-07-12T00:00:00+00:00
[2023-07-22T05:56:54.025+0000] {base.py:71} INFO - Using connection ID 'postgres' for task execution.
[2023-07-22T05:56:54.058+0000] {sql.py:315} INFO - Running statement: 
            CREATE TABLE IF NOT EXISTS users (
                firstname TEXT NOT NULL,
                lastname TEXT NOT NULL,
                country TEXT NOT NULL,
                username TEXT NOT NULL,
                password TEXT NOT NULL,
                email TEXT NOT NULL
            );
        , parameters: None
[2023-07-22T05:56:54.071+0000] {postgres.py:96} INFO - NOTICE:  relation "users" already exists, skipping

[2023-07-22T05:56:54.098+0000] {taskinstance.py:1406} INFO - Marking task as SUCCESS. dag_id=user_processing, task_id=create_table, execution_date=20230712T000000, start_date=20230722T055653, end_date=20230722T055654
[2023-07-22T05:56:54.296+0000] {local_task_job.py:164} INFO - Task exited with return code 0
[2023-07-22T05:56:54.436+0000] {local_task_job.py:273} INFO - 1 downstream tasks scheduled from follow-on schedule check
